src/ontology/foodon-edit.ofn

```diff
@@ -20657,6 +20660,32 @@ AnnotationAssertion(rdfs:label obo:FOODON_00004502 "fish (raw)"@en)
 EquivalentClasses(obo:FOODON_00004502 ObjectIntersectionOf(obo:FOODON_03411222 ObjectSomeValuesFrom(obo:RO_0000053 obo:FOODON_00004348)))
 SubClassOf(obo:FOODON_00004502 obo:FOODON_03411222)
 
+# Class: obo:FOODON_00004503 (wild animal)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004503 "An undomesticated animal specie."@en)
```

Comment:
Singular of "species" is actually still "species".  Also, it isn't necessarily that wild animals are different species wise from domestic ones.  Wild horses for example, or wild turkeys.  So you could probably just leave the "species" part out.